docs/requirements.md

```diff
@@ -53,3 +53,4 @@ Notes:
 
 - To lower costs, you can shut down your server when you're not using it.
 - We recommend using the `gcloud cli` to avoid using the GCP Dashboard if possible.
+- To enable HTTPS, you have to add an external domain.
```

Comment:
Is this strictly true? In air-gapped networks, you can use self-signed certificates or certificates signed by an internal certificate authority, and if you control DNS then everything is OK.

I think this statement is fine but would suggest we soften it to "For serving code-server over HTTPS, we recommend using an external domain name along with a service such as Let's Encrypt"

Reply:
Ahhh okay, I didn't know. This came after a GitHub Discussion. Let's go with your wording instead.